SQL/New_patches/2020-01-16-ModuleManager.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO modules (Name, Active) VALUES ('module_manager', 'Y');
+INSERT INTO permissions (code, description, categoryID) VALUES('module_manager_view', 'Access the module_manager module', 2);
```

Comment:
Do you think you could change these to something like 'Module manager: View the module' and 'Module manager: Edit entries within the module'?

I find it useful to have permissions have $ModuleName at the front of them, even though this isn't standard.

Reply:
Updated them